feat (model): local hugging face cache dir

wangsrGit119 opened the pull request:
1.  Update README.md

reycn commented:
If I recall correctly, `hf_hub_download` auto-detects `HF_HOME`. To be verified later.

wangsrGit119 commented:
In the last update, I have deleted this new piece, and I can merge the others.
In addition, HF_HOME is in the user directory by default, and I have closed the pr submitted yesterday.

deleted code link.
https://github.com/Byaidu/PDFMathTranslate/pull/147/commits/ccce2a55e379e33eb6f9002dd122b98dd52ccfbc
yesterday pr.
https://github.com/Byaidu/PDFMathTranslate/pull/143

Byaidu commented:
dev的内容写在install不太合适，这方面我们之后会写在contributing guide里 https://github.com/Byaidu/PDFMathTranslate/issues/145
